hs_core/hydroshare/resource.py

```diff
@@ -77,7 +77,11 @@ def get_science_metadata(pk):
     Exceptions.NotFound  - The resource identified by pid does not exist
     Exception.ServiceFailure  - The service is unable to process the request
     """
-    return get_system_metadata(pk)
+    res = utils.get_resource_by_shortkey(pk)
+    if res:
+        return res.metadata.get_xml()
+
+    #return get_system_metadata(pk)
```

Comment:
@ironfroggy - While REST API work was just getting under way having a Dockerfile associated to that work that doesn't collide with anyone else's work and is an easy way to figure out which packages need to be brought in prior to merging the code.

- Dockerfile (Dockerfile.local) - used in generic non-iRODS development environment
- Dockerfile.irods - used when running in a production or development iRODS environment
- Dockerfile.djangorest - used in REST development environment

usage: $ cp Dockerfile.{local, irods, djangorest} Dockerfile

Then build for your environment.